chanjo_report/server/blueprints/report/templates/report/gene.html

```diff
@@ -66,7 +66,7 @@ <h1>Gene: {{ gene_name or gene_id }}</h1>
 													{% else %}
 														{{ exon.chrom }}:{{ exon.start }}-{{ exon.end }}
 													{% endif %}
-													<span class="pull-right">
+													<span class="float-right">
```

Comment:
So basically this becomes a POST request where the gene list is passed as an input, which is wise, given that it might be a long list.

The endpoint views.genes already accepts the gene list from post requests, so everything is fine in the code!

![image](https://user-images.githubusercontent.com/28093618/80124330-68e75f80-8590-11ea-8dab-12ceea27880c.png)